libs/global-constants/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@mdn/global-constants",
```

Comment:
I can't be constructive with this but, **can we use a different naming convention?**
To me, this `@mdn/...` makes me think of the npmjs.com `@mdn` organization. 
But there exists no `https://www.npmjs.com/package/@mdn/global-constants` and that might be confusing as hell to someone reading the `const { VALID_LOCALES } = require("@mdn/global-constants");` line. 

What about something specific that helps us remember that this is a trick/hack so we can have code in this monorepo built into sub-packages by disk. I don't want to start a bikeshedding but some crazy ideas:

* `yari/global-constants`
* `yari-shared/global-constants`
* `yari-shared/constants` (my favorite)
* `yari-globals/constants`
* `yari-shared-globals/constants`
* `yari-shared-lib/constants`
* `yari-lib/constants`

I'm not feeling super strong on this but I think it's worth taking the time to discuss before this sets in stone and becomes a naming thing we have to really stick to.

Reply:
I like `yari-shared/constants` and `yari-shared/fundamental-redirects`, but I don't feel strongly about this either.

Reply:
I would propose `yari-internal` as they are not published, how does that sound?

Reply:
👍 I like `yari-internal`.